codegen/templates/grpc_client.tmpl

```diff
@@ -200,16 +200,16 @@ func (e *{{$clientName}}) {{$methodName}}(
 			opts = append(opts, yarpc.WithHeader(e.opts.RequestUUIDHeaderKey, reqUUID))
 		}
 	}
-	ctx, cancel := context.WithTimeout(ctx, e.opts.Timeout)
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, e.opts.Timeout)
```

Comment:
can we leave a comment here why ctx was not reused instead a new var ctxWithTimeout was declared? Someone else may make some changes and make a mistake here

Reply:
sure

Reply:
done